### ISSUE TITLE
small glitch - Walkable hovering chair behind Guthbered

### ISSUE THREAD

**Issue by GoogleCodeExporter** [alistairreilly/andors-trail#204]
```
if you chose to kill Guthbered and you kill him you can go to the chair behind 
him and your character is not showing over the chair.
```

Original issue reported on code.google.com by `dope...@gmail.com` on 2 Jul 2011 at 3:57

**Comment by GoogleCodeExporter** [comment]
```
Thank you for the report. Yes, this is a bug. 

This is now fixed in r121. Marking this as ready for the next release.
```

Original comment by `oskar.wi...@gmail.com` on 8 Jul 2011 at 3:51
- Added labels: **Milestone-0.6.10**

**Comment by GoogleCodeExporter** [comment]
Original comment by `oskar.wi...@gmail.com` on 13 Jul 2011 at 8:27
- Changed state: **Ready**

**Comment by GoogleCodeExporter** [comment]
Original comment by `oskar.wi...@gmail.com` on 16 Jul 2011 at 10:12
- Changed title: **small glitch - Walkable hovering chair behind Guthbered**

**Comment by GoogleCodeExporter** [comment]
```
Released in v0.6.10 . Closing.
```

Original comment by `oskar.wi...@gmail.com` on 21 Oct 2011 at 8:56
- Changed state: **Fixed**
